modules/candidate_parameters/php/module.class.inc

```diff
@@ -33,4 +18,131 @@ class Module extends \Module
     {
         return "Candidate Parameters";
     }
+
+    /**
+     * {@inheritDoc}
+     *
+     * @param string $type    The type of widgets to get.
+     * @param \User  $user    The user widgets are being retrieved for.
+     * @param array  $options A type dependent list of options to provide
+     *                        to the widget.
+     *
+     * @return \LORIS\GUI\Widget[]
+     */
+    public function getWidgets(string $type, \User $user, array $options) : array
+    {
+        switch($type) {
+        case 'candidateinfo':
+            // We add 3 extra types of cards to the candidate info card on
+            // the candidate dashboard based on the default LORIS candidate_parameter
+            // module:
+            //
+            // 1. The Participant Status
+            // 2. The "Candidate Parameters" category variables which are displayed
+            //    in the Candidate Information tab of the Candidate Parameters page
+            // 3. The Caveat flag description/reason for the candidate. (Only if
+            //    there is a caveat.)
+            //
+            // Most other tabs in the candidate_profile page are either duplicates
+            // of the information already present (ie. DoB) or complicated enough to
+            // merit their own card on the dashboard.
+            $factory = \NDB_Factory::singleton();
+            $db      = $factory->database();
+
+            $candidate = $options['candidate'];
+            if ($candidate === null) {
+                return [];
+            }
+
+            // Participant Status
+            $entries = [
+                new CandidateInfo(
+                    "Participant Status",
+                    $candidate->getParticipantStatusDescription($db)
```

Comment:
```suggestion
                    $candidate->getParticipantStatusDescription($db) ?? 'N/A'
```
Or something similar. Participant Status is mandatory if the value is change in the front-end but there are cases where it can be blank (at least in Raisinbread).
<img width="502" alt="Screen Shot 2020-03-03 at 15 23 27" src="https://user-images.githubusercontent.com/4022790/75816408-21e4a700-5d63-11ea-8509-21912cf7677c.png">